app/models/user.rb

```diff
@@ -11,13 +12,5 @@ class User < ApplicationRecord
 
   serialize :preferences, coder: JsonSerializer
 
-  def full_name
-    "#{first_name} #{last_name}".strip.presence
-  end
-
-  def initials
-    return nil if first_name.blank? || last_name.blank?
-
-    first_name[0].capitalize + last_name[0].capitalize
-  end
+  normalizes :first_name, :last_name, with: -> attribute { attribute.strip.capitalize }
```

Comment:
I think this will mess up first names such as “TJ” and last names such as “McMaster”